subtests/redhat/rhel_push_plugin/rhel_push_plugin.py

```diff
@@ -24,6 +24,10 @@ class rhel_push_plugin(subtest.SubSubtestCaller):
 
     """ SubSubtest caller """
 
+    def initialize(self):
+        self.failif_not_redhat()
+        super(packaging, self).initialize()
```

Comment:
Hrmmm, there's a subtle problem here.  I was hoping to have these tests not contribute toward a FAIL (in CI).  However, top-level TestNAErrors do have that effect :-1:  But moving this call into the sub-subtests feels wrong to me.